docs/pages/setup/reference/cli.mdx

```diff
@@ -311,6 +311,39 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy app
+
+Start a local TLS proxy for Application Access connections when using Teleport in single-port mode.
+
+```code
+$ tsh proxy app [<flags>] <app>
+```
+
+#### Arguments
+`<app>`
+
+- `app` The name of the application to start the local proxy for. To see a list of available applications, run `tsh apps ls`.
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--port` | none | port | Specify the source port for the local proxy |
+
+#### [Global Flags](#tsh-global-flags)
+
+These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
+Run `tsh help <subcommand>` or see the [Global Flags Section](#tsh-global-flags)
+
+#### Examples 
+```code
+$ tsh --proxy proxy.example.com:443 proxy app <app>
+
+# Proxy a connection to drone app on local port 10700
+$ tsh proxy --port 10700 app drone
```

Comment:
drone is probably a bit obscure.  I'd lean towards jenkins or grafana here.  Also, can we add a follup curl or api call to `127.0.0.1:10700` showing how to use this proxy?

Reply:
Do you mean the followup curl or API call as additional output from `tsh proxy --port 10700 app drone`. Or just documentation added here?

Reply:
Changed example from drone to grafana with https://github.com/gravitational/teleport/pull/10924/commits/271d714d4f51bbe6b7ff1988384fa90ce37c6955

Reply:
> Do you mean the followup curl or API call as additional output from `tsh proxy --port 10700 app drone`. Or just documentation added here?

See this suggestion: https://github.com/gravitational/teleport/pull/10924#discussion_r822194739